.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   build_and_run:
-    runs-on: ubuntu-8
+    runs-on: ubuntu-latest
```

Comment:
I'm not familiar with the TEE set up but could this break the environment with unexpected changes in OS versions?

Reply:
This is just a change to the CI of the nitro-testnode repo. I think this was a change brought in by a previous merge. I'm not even sure if github has ubuntu 8 runners anymore. The CI on this PR passed with the ubuntu 24 runner, so it should be fine, but I can try changing it back.

Reply:
ahhh if it's on the nitro CI then we usually just stick with that right? if so, ignore my comment